promgen/models.py

```diff
@@ -599,6 +607,15 @@ def highlight(self):
             return "danger"
         return ""
 
+    @staticmethod
+    def find_parent_object(obj):
```

Comment:
I think specifying `_object` is redundant. And also, `get` is more appropriate than `find` in this case.

I would name this `get_parent`, as simple as that.

Later in the code, there are also a few more redundant things like:
- `instance_parent_object` -> `parent`
- `old_parent_object` -> `old_parent`

Please apply this patch (`git am <that file>`): 
[0001-fixup-Modify-the-Audit-model-to-improve-the-Audit-Lo.PATCH](https://github.com/user-attachments/files/21747712/0001-fixup-Modify-the-Audit-model-to-improve-the-Audit-Lo.PATCH)